bpf/lib/identity.h

```diff
@@ -6,6 +6,21 @@
 
 #include "dbg.h"
 
+#define LOCAL_CLUSTER_IDENTITY_RANGE_START 1
+#define LOCAL_CLUSTER_IDENTITY_RANGE_END   ((1 << 16) - 1)
```

Comment:
Not sure if it may be a good idea to define these in `pkg/identity`? (initially I thought I would find these ranges already defined in there, but so far I could only find them in the docs)